internal/locale/locales/en-us.yaml

```
@@ -994,10 +994,14 @@ field_bitwidth:
   other: Bit Width
 commit_message_add_platform:
   other: "Added platform: {{.V0}} {{.V1}}bit {{.V2}}"
+commit_message_updated_platform:
+  other: "Updated platform: {{.V0}} to {{.V1}}"
 commit_message_removed_platform:
   other: "Removed platform: {{.V0}} {{.V1}}bit {{.V2}}"
 commit_message_add_language:
   other: "Added language: {{.V0}} {{.V1}}"
+commit_message_updated_language:
+  other: "Updated language: {{.V0}} {{.V1}}"
```

```suggestion
  other: "Updated language: {{.V0}} to {{.V1}}"
```
Just to keep this in line with the other `updated` message